### PR TITLE
Make logging level configurable.

### PR DESCRIPTION
**Pull request by atoulme**
Add a bit of code to log error stack trace - that could be improved upon, let me know.

### CODE REVIEW

File: src/cli/commands/beacon.ts [hunk 1]
```diff
@@ -15,12 +15,14 @@ import deepmerge from "deepmerge";
 import {getTomlConfig, IConfigFile} from "../../util/toml";
 import defaults from "../../node/defaults";
 
+
```
Comment (GregTheGreek) [inline src/cli/commands/beacon.ts]: Can you please remove whitespace :)

**Review by GregTheGreek — CHANGES_REQUESTED** [review]

File: src/cli/commands/beacon.ts (hunk 1 above)
Comment (atoulme) [reply src/cli/commands/beacon.ts]: done

**Review by GregTheGreek — APPROVED** [review]

File: src/cli/commands/beacon.ts [hunk 2]
```diff
@@ -34,18 +35,23 @@ export class BeaconNodeCommand implements CliCommand {
       .option("-eth1, --eth1RpcUrl [url]", "Url to eth1 rpc node")
       .option("--rpc [api]", "Exposes the selected RPC api, must be comma separated")
       .option("-c, --configFile [config_file]", "Config file path")
+      .option("-l, --loggingLevel [DEBUG|INFO|WARN|ERRROR]", "Logging level")
```
Comment (GregTheGreek) [inline src/cli/commands/beacon.ts]:
```suggestion
      .option("-l, --loggingLevel [DEBUG|INFO|WARN|ERROR]", "Logging level")
```

**Review by GregTheGreek — CHANGES_REQUESTED** [review]

**Review by wemeetagain — APPROVED** [review]

**Review by GregTheGreek — APPROVED** [review]

File: src/cli/commands/beacon.ts [hunk 3]
```diff
@@ -34,18 +35,23 @@ export class BeaconNodeCommand implements CliCommand {
       .option("-eth1, --eth1RpcUrl [url]", "Url to eth1 rpc node")
       .option("--rpc [api]", "Exposes the selected RPC api, must be comma separated")
       .option("-c, --configFile [config_file]", "Config file path")
+      .option("-l, --loggingLevel [DEBUG|INFO|WARN|ERROR]", "Logging level")
```
Comment (mpetrunic) [inline src/cli/commands/beacon.ts:38]: maybe generate options from LogLevel enum instead of hardcoding?

**Review by mpetrunic — APPROVED** [review]

File: src/cli/commands/beacon.ts (hunk 3 above)
Comment (GregTheGreek) [reply src/cli/commands/beacon.ts:38]: Subsequent PR :)

Comment (atoulme) [reply src/cli/commands/beacon.ts:38]: See #218

**Pull request merged**